__tests__/integration/mint-tokens.test.js

```diff
@@ -255,25 +266,87 @@ describe('mint token', () => {
     expect(p2pkh.address.base58).toEqual(address0);
   });
 
+  it('should mint and create data outputs', async () => {
+    const response = await TestUtils.request
+      .post('/wallet/mint-tokens')
+      .send({
+        token: tokenA.uid,
+        amount: 100,
+        data: ['foobar1', 'foobar2'],
+      })
+      .set({ 'x-wallet-id': wallet1.walletId });
+
+    // Current funds: 9 HTR + 960 TKA
```

Comment:
Why 9 HTR if you created two data outputs?

Reply:
Youre correct, forgot to remove the 2 htr